devicetypes/smartthings/zigbee-window-shade.src/zigbee-window-shade.groovy

```diff
@@ -82,9 +81,19 @@ private List<Map> collectAttributes(Map descMap) {
 	return descMaps
 }
 
+def installed() {
```

Comment:
You should rebase on the latest `master` because the change to the DTH to create the `supportedWindowShadeCommands` at install time was already added.